policy/modules/system/init.te

```diff
@@ -755,6 +756,7 @@ fs_unmount_all_fs(initrc_t)
 fs_remount_all_fs(initrc_t)
 fs_getattr_all_fs(initrc_t)
 fs_search_all(initrc_t)
+fs_search_all(initrc_state_t)
```

Comment:
`fs_search_all(initrc_state_t)` doesnt make sense. initrc_state_t is a file type, domains do the searching, files don't.

This looks like parts were merged already? Is this just a stray line that I dropped when I merged the rest? It looks like there are a bunch of other commits in this PR so I'm guessing it picked up more when I merged stuff?

This looks like the only outstanding line? Can you re-check and fix this PR so its just the needed stuff or maybe close if we've already merged the important bits?